nexus/src/app/background/driver.rs

```diff
@@ -88,17 +98,33 @@ impl Driver {
         imp: Box<dyn BackgroundTask>,
         opctx: OpContext,
         watchers: Vec<Box<dyn GenericWatcher>>,
-    ) -> TaskHandle {
+        activator: &Activator,
+    ) -> TaskName {
         // Activation of the background task happens in a separate tokio task.
         // Set up a channel so that tokio task can report status back to us.
         let (status_tx, status_rx) = watch::channel(TaskStatus {
             current: CurrentStatus::Idle,
             last: LastResult::NeverCompleted,
         });
 
-        // Set up a channel so that we can wake up the tokio task if somebody
-        // requests an explicit activation.
-        let notify = Arc::new(Notify::new());
+        // We'll use a `Notify` to wake up that tokio task when an activation is
+        // requested.  The caller provides their own Activator, which just
+        // provides a specific Notify for us to use here.  This allows them to
+        // have set up the activator before the task is actually created.
```

Comment:
Sure, I'll remove this sentence and mention this in the doc comment.  (I'm mostly going to lean on the `init` module-level docs though, rather than explain it again here.)

Reply:
Yeah...this comment is partially a result of my reviewing the files in order and not having gotten to that comment yet. Once I got that far, I kinda forgot I had left this here...maybe a "see the comments in the `init` module" would suffice?